scala/README.md

```diff
@@ -154,27 +165,31 @@ Please read the [Scala Style Guide] carefully. The main points to consider are:
     ```scala
     package foo.bar
 
-    // bad
+    // Bad
     import baz.Qux
 
-    // good
+    // Good
     import foo.bar.baz.Qux
     ```
 
+0. Avoid wildcard-importing entire packages: `import package._`
+
 0. The bigger the scope, the more descriptive the name. Only for very small, local scopes may single-letter mnemonics be used.
 
 0. Use `_` for simple, single line functions:
 
     ```scala
-    seq filter (_ % 2 == 0)
+    val bad = seq filter (number => number % 2 == 0)
+
+    val good = seq filter (_ % 2 == 0)
```

Comment:
can't remember, do we mention somewhere in this document to use `()` over `{}` for simple single line functions?